segmentation/utils/spatial_analysis_test.py

```diff
@@ -14,7 +14,7 @@ def make_threshold_mat():
     return thresh
 
 
-def make_distance_matrix(enrichment_type):
+def make_distance_matrix(enrichment_type, analysis_type="spatial"):
```

Comment:
Noticed that the analysis_type argument isn't used in the function make_distance_matrix nor make_expression_matrix. Just curious if you meant to add functionality for this later or if it was a parameter you ended up not needing.